drafts/draft-ietf-ippm-ioam-direct-export.xml

```diff
@@ -300,38 +305,98 @@ packets   |Encapsu-|     | Transit|     | Transit|     |Decapsu-|
-                      with the DEX option MAY export the requested
-                      IOAM data, and MUST decapsulate the IOAM header.</t>
+                      with the DEX option MAY export and/or collect the 
+					  requested IOAM data, and MUST decapsulate the IOAM 
+					  header.</t>
           </list></t>
 
-		  <t>As in <xref target="I-D.ietf-ippm-ioam-data"/>, the DEX option 
-		  may be incorporated into all or a subset of the traffic that is 
-		  forwarded by the encapsulating node. Moreover, IOAM nodes MAY
-		  export data for all traversing packets that carry the DEX 
-		  option, or MAY selectively export data only for a subset of these 
-		  packets.</t>
+		  <t>As in <xref target="I-D.ietf-ippm-ioam-data"/>, the DEX option
+		  can be incorporated into all or a subset of the traffic that is
+		  forwarded by the encapsulating node, as further discussed in
+		  <xref target="SelectionSec"/> below. Moreover, IOAM nodes respond
+		  to the DEX trigger by exporting and/or collection IOAM data
+		  either for all traversing packets that carry the DEX
+		  option, or selectively only for a subset of these
+		  packets, as further discussed in <xref target="ExportSec"/> below.</t>
 		  
-		  <t>The DEX option specifies which data fields should be exported, 
-		  as specified in <xref target="OptionSec"/>.
-		  The format and encapsulation of the packet that contains the exported 
-		  data is not within the scope of the current document. For example, 
-		  the export format can be based on 
+		  <section anchor="SelectionSec" title="DEX Packet Selection">
+
+		  <t>If an IOAM encapsulating node incorporates the DEX option into all
+		  the traffic it forwards it may lead to an excessive amount of exported
+		  data, which may overload the network and the receiving entity.
+		  Therefore, an IOAM encapsulating node that supports the DEX option MUST 
+		  support the ability to incorporate the DEX option selectively
+		  into a subset of the packets that are forwarded through it.</t>
+
+		  <t>Various methods of packet selection and sampling have been previously
+		  defined, such as <xref target="RFC7014"/> and <xref target="RFC5475"/>.
+		  Similar techniques can be applied by an IOAM encapsulating node to
+		  apply DEX to a subset of the forwarded traffic.</t>
+
+		  <t>The subset of traffic that is forwarded or transmitted with a DEX
+		  option SHOULD not exceed 1/N of the interface capacity on any of
+		  the IOAM encapsulating node's interfaces. It is noted that this
+		  requirement applies to the total traffic that incorporates a DEX option,
+		  including traffic that is forwarded by the IOAM encapsulating node and
+		  probe packets that are generated by the IOAM encapsulating node.
+		  In this context N is a parameter that can be configurable by network
+		  operators. If there is an upper bound, M, on the number of
+		  IOAM transit nodes in any path in the network, then it is RECOMMENDED
```

Comment:
s/RECOMMENDED/recommended/

RECOMMENDED is not 2119 language. Suggest to avoid uppercase.

Reply:
Updated. BTW, RECOMMENDED is in the RFC 2119 language.